yang/vyatta-protocols-frr-bgp-v1.yang

```diff
@@ -1626,10 +1776,33 @@ module vyatta-protocols-frr-bgp-v1 {
 						uses parameters-distance;
 						uses parameters-dampening;
 					}
+          container label {
+            description "label value for VRF";
+            configd:help "label value for VRF";
+            container vpn {
+              description "Between current address-family and vpn";
+              configd:help "Between current address-family and vpn";
+              leaf export {
+                description "For routes leaked from current address-family to vpn";
+                configd:help "For routes leaked from current address-family to vpn";
+                type uint32 {
```

Comment:
why is this needed ? bgp should allocate the vpn label. The "label vpn export auto" configuration assigns a vrf label ? Why can't this be done instead of using a static label ?